RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -145,13 +145,14 @@ public class PaywallViewController: UIViewController {
 
         self.shouldBlockTouchEvents = shouldBlockTouchEvents
         self.dismissRequestedHandler = dismissRequestedHandler
+        let handler = PurchaseHandler.default()
 
         self.configuration = .init(
             content: content,
             mode: Self.mode,
             fonts: fonts,
             displayCloseButton: displayCloseButton,
-            purchaseHandler: purchaseHandler
+            purchaseHandler: handler
```

Comment:
whoa, this is not correct

Reply:
Oh yeah, sorry, I missed the part above. I merged the two behaviors.